tools/linters/go.mod

```diff
@@ -169,3 +170,10 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
 )
+
+// TODO: Remove once golangci is updated.
```

Comment:
This is better tracked as a ticket in the backlog, for visibility so anyone can follow up.

Reply:
I'll make one, but I didn't bother at first because we have dependabot running weekly, so I figured whoever reviews the PR which updates `github.com/golangci/golangci-lint` would notice the TODO commented above it. I should also clarify it won't be fixed until `> 1.50.1`